src/synopsys-action/synopsys-bridge.ts

```diff
@@ -28,7 +29,8 @@ export class SynopsysBridge {
     this.bridgeExecutablePath = ''
     this.synopsysBridgePath = ''
     this.bridgeArtifactoryURL = 'https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/synopsys/integration/synopsys-bridge/'
```

Comment:
we can add bridgeArtifactoryURL in application-constants.ts as BRIDGE_ARTIFACTORY_URL